docs/source/1.0/spec/core/traits.rst

```diff
@@ -138,10 +138,12 @@ trait is defined in the IDL by enclosing the value in parenthesis. Trait values
 can only appear immediately before a shape.
 
 .. productionlist:: smithy
-    trait                   :"@" `shape_id` ["(" `trait_body_value` ")"]
-    trait_body_value        :`trait_structure` / `node_value`
-    trait_structure         :`trait_structure_kvp` *("," `trait_structure_kvp`)
-    trait_structure_kvp     :`text` ":" `node_value`
+    trait_statements    : *(`ws` `trait`) `ws`
+    trait               :"@" `shape_id` [`trait_body`]
+    trait_body          :"(" `ws` `trait_body_value` `ws` ")"
+    trait_body_value    :`trait_structure` / `node_value`
+    trait_structure     :`trait_structure_kvp` *(`ws` `comma` `trait_structure_kvp`)
```

Comment:
This should probably have a `ws` after the `comma`

Reply:
`comma` includes `ws` in the production:

```
comma = "," ws
```